src/logic/synchronization.ts

```diff
@@ -63,23 +64,27 @@ async function exportMetadata(d2: D2, builder: ExportBuilder): Promise<Synchroni
 async function importMetadata(
     d2: D2,
     instanceId: string,
-    metadataPackage: SynchronizationResult
-): Promise<void> {
-    // TODO: Obtain import results and add warnings/errors to the logger
+    metadataPackage: MetadataPackage
+): Promise<SynchronizationResult> {
     const instance = await Instance.get(d2, instanceId);
     const importResult = await postMetadata(instance, metadataPackage);
 
-    // DEBUG: Print response from import
-    console.log("[DEBUG] Destination instance", instance);
-    console.log("[DEBUG] Metadata package", metadataPackage);
-    console.log("[DEBUG] HTTP Status", importResult.status);
-    console.log("[DEBUG] Import Stats", importResult.data.status, importResult.data.stats);
-    // END OF DEBUG SECTION
+    // TODO: Update the logger
 
-    return importResult;
+    return {
+        ...importResult.data,
+        instance: {
+            id: instance.id,
+            name: instance.name,
+            url: instance.url,
+        },
```

Comment:
It's ok like this. When the number of properties grow: `_.pick(instance, ["id", "name", "url", ...])`